2017-2018/GameEngine/comm/stateConverter.py

```diff
@@ -23,6 +23,17 @@ def _parse_grid_element(cls, el):
             return pacmanState_pb2.PacmanState.EMPTY
 
     @classmethod
+    def _parse_direction(cls, direction):
```

Comment:
It might make more sense to implement this as a lookup into a constant dictionary? I believe that's technically more efficient.

Reply:
Makes sense +1